#Goal Definition
geometry_msgs/PoseStamped goal
